src/hid_freebsd.c

```diff
@@ -9,6 +9,11 @@
 #include <dev/usb/usb_ioctl.h>
 #include <dev/usb/usbhid.h>
 
+#if defined(_HID_HID_H_)
+#define	HAS_HIDRAW
+#include <dev/hid/hidraw.h>
+#endif
```

Comment:
Would it be more robust to check `__FreeBSD__` and `__FreeBSD_version`, or have CMake check if `dev/hid/hidraw.h` exists?

Reply:
Good point, `__FreeBSD_version` is the easiest way. Checking for `__FreeBSD__` probably unnecessary at that point, as CMakefile only builds that file if building on/for FreeBSD.